3684/0/about_sandwich_code.rb

```diff
@@ -1,13 +1,17 @@
 require File.expand_path(File.dirname(__FILE__) + '/neo')
 
+# This method smells of :reek:NilCheck
+# This method smells of :reek:UtilityFunction
+# This method smells of :reek:UncommunicativeMethodName
+
 class AboutSandwichCode < Neo::Koan
   def count_lines(file_name)
     file = File.open(file_name)
     count = 0
     count += 1 while file.gets
     count
   ensure
-    file.close
+    file&.close
```

Comment:
please change this. implement a guard clause above the `ensure` block and remove `nilcheck` here. after that remove :reek:NilCheck

Reply:
outdated